blocks/core/ff_container/ff_container-dialog/ff_container-dialog.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+var React = require('react');
+
+function generateClass(base, props){
+    var classNames = [];
+        classNames.push(base);
+        if (!!props.modifier) classNames.push(base + '--' + props.modifier);
```

Comment:
I know the distinction there, I'm just saying that there were some actual values values for modifier and classes that produced false positives without the doublebang, though as I say I can't recall the exact one.